docs/react-integration.md

```diff
@@ -172,31 +175,38 @@ The following props will be given to your wrapped component:
  - `fetchStatus`: the status of the fetch (`pending`, `loading`, `loaded` or `error`)
  - `lastFetch`: when the last fetch occurred
  - `hasMore`: the fetches being paginated, this property indicates if there are more documents to load
+ - `isFetching`: a boolean that you can use to show that it's in a fetching state, regardless of the state of the `fetchStatus` variable.
+   Because sometimes you may want to display an additional indicator that a query is refetching in the background. 
+   And query's `fetchStatus === 'loading'` state is not enough.
 
 
 You can also pass a function instead of a direct query definition. Your function will be given the props of the component and should return the requested query definition:
 
 ```jsx
 import { Query, Q } from 'cozy-client'
+import React from 'react'
 
-const queryDefinition = function (props) {
+const queryDefinition = function(props) {
   const todos = 'io.cozy.todos'
   const where = { checked: props.checked }
   return Q(todos).where(where)
 }
 
-function TodoList() {
-  return <Query query={queryDefinition} checked={props.checked}>
-    {
-      ({ data, fetchStatus }) =>
+const TodoList = ({ props }) => (
+  <Query query={queryDefinition} checked={props.checked}>
+    {({ data, fetchStatus, isFetching }) => (
       fetchStatus !== 'loaded'
         ? <h1>Loading...</h1>
-        : <ul>{data.map(todo => <li>{todo.label}</li>)}</ul>
+        : <>
+          {isFetching && <h1>Refreshing data...</h1>}
```

Comment:
See my previous comment. Since the QDef is not talking about backgroundSync, isFetching should not be set to true